Formula/quickjs.rb

```diff
@@ -1,22 +1,32 @@
 class Quickjs < Formula
   desc "Small and embeddable JavaScript engine"
   homepage "https://bellard.org/quickjs/"
-  url "https://bellard.org/quickjs/quickjs-2019-07-09.tar.xz"
-  sha256 "350c1cd9dd318ad75e15c9991121c80b85c2ef873716a8900f811554017cd564"
+  url "https://bellard.org/quickjs/quickjs-2019-07-21.tar.xz"
+  sha256 "a906bed24c57dc9501b84a5bb4514f7eac58db82b721116ec5abe868490e53cc"
 
   bottle do
     sha256 "989215f264fc3240904342524457b923da278d6edd53e8fb44dad1f370b4e64f" => :mojave
     sha256 "0f472d5f083df02c3516f42c43333f09fd6e2734da64ce7a4ae027a7e4dc808f" => :high_sierra
     sha256 "3fbf3dfc902c9e68415088325ec7ec6f16e1c69204b3cd3432046d6bed14896e" => :sierra
   end
 
+  patch :DATA
+
   def install
-    system "make", "install", "prefix=#{prefix}"
+    system "make", "prefix=#{prefix}", "CONFIG_M32="
+    # Tests are dependent on having a TTY, so fake it with `script`
+    system "script", "-q", "/dev/stdout", "make", "test", "prefix=#{prefix}", "CONFIG_M32="
```

Comment:
Why is `make test` needed?

Reply:
It's not needed but I thought it's a good idea to run the tests built into the library. Unfortunately they have to be ran at build stage because they are implemented in the Makefile and build modules themselves.

Reply:
We don't run build-time tests in Homebrew (except for some security software). Please remove this.

Reply:
Removed